<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Queries</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidgetQueryEditor">
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Welcome</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Ctrl + N for new Query Editor</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>603</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnections">
    <property name="title">
     <string>Connections</string>
    </property>
    <addaction name="actionNewConn"/>
   </widget>
   <widget class="QMenu" name="menuApp">
    <property name="title">
     <string>App</string>
    </property>
    <addaction name="actionExitApp"/>
   </widget>
   <widget class="QMenu" name="menuQueries">
    <property name="title">
     <string>Queries</string>
    </property>
    <addaction name="actionNewQuerySession"/>
   </widget>
   <addaction name="menuApp"/>
   <addaction name="menuConnections"/>
   <addaction name="menuQueries"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewConn">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionExitApp">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNewQuerySession">
   <property name="text">
    <string>New Session</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionNewConn</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openNewConnDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExitApp</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNewQuerySession</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>newQuerySessionTab()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openNewConnDialog()</slot>
  <slot>selectedConnChanged(int)</slot>
  <slot>executeQuery()</slot>
  <slot>newQuerySessionTab()</slot>
 </slots>
</ui>
